libs/datamodel/core/src/transform/ast_to_dml/db/context.rs

```diff
@@ -18,6 +18,12 @@ pub(crate) struct Context<'ast> {
     pub(super) diagnostics: Diagnostics,
     arguments: Arguments<'ast>,
     attributes: Attributes<'ast>,
+
+    // @map'ed names indexes. These are not in the db because they are only used for validation.
+    pub(super) mapped_model_names: HashMap<&'ast str, ast::ModelId>,
```

Comment:
should these be one level down in a struct instead?

Reply:
We can group them, yes, it's just a bit more indirection for the reader. I'll try that.